### PR TITLE
Update ARTWORK_LICENSE

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Reflow file to allow machine parsing. Correct typos and missing paths.
Remove nonexistent file uninhabited-jungle.png

### CODE REVIEW

File: dat/gfx/ARTWORK_LICENSE [hunk 1]
```diff
@@ -1142,9 +1124,8 @@
        * uninhabited-rocky-magenta-03.png
        * uninhabited-rocky-magenta-04.png
 
- * Author: Fernando L. Copercini "Bisc88" <fernandoluis1732@yahoo.com.br>, License: GPLv3+
+ * Author: Fernando L. Copercini "Bisc88" <fernandoluis1732@yahoo.com.br>, License: GPLv3
```
Comment (ids1024) [inline dat/gfx/ARTWORK_LICENSE]: Why remove the + from GPLv3+ here? These are distinct licensing terms. Software licensed GPLv3+ can also be used under the terms of version 4 or later of the GPL, if GNU ever releases another version.

File: dat/gfx/ARTWORK_LICENSE [hunk 2]
```diff
@@ -1142,9 +1124,8 @@
...
    * planet
      * exterior
        * snowy_cliffs.png
        * rocky_small.png
-       * uninhabited-jungle.png
```
[the remark this answers is not in the document]
Comment (FedericoCeratto) [reply dat/gfx/ARTWORK_LICENSE]: The file is missing in the 0.7.0 tarball. I'll update.

File: dat/gfx/ARTWORK_LICENSE (hunk 1 above)
Comment (FedericoCeratto) [reply dat/gfx/ARTWORK_LICENSE]: It looked like a mistake as all other files are under GPLv3 but I found the commit from the author.

File: dat/gfx/ARTWORK_LICENSE [hunk 3]
```diff
@@ -918,20 +918,12 @@
 
  * Author: Unnamed, License: CC-by-sa 3.0
    * outfit
-     * space
```
Comment (ids1024) [inline dat/gfx/ARTWORK_LICENSE]: Likewise, these shouldn't be removed. I think all the files here are in git.

**Comment by onpon4** [comment]
I've implemented most of the proposed changes in this PR, excluding removals (which as @ids1024 mentioned isn't something we want), via d855c9f.

**Pull request closed without merging**